.gitlab-ci.yml

```diff
@@ -0,0 +1,65 @@
+variables:
+  ALPAKA_CI_OS_NAME: "Linux"
+  ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: "OFF"
+  ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE: "OFF"
+  ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE: "ON"
+  ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE: "ON"
+  ALPAKA_ACC_ANY_BT_OMP5_ENABLE: "OFF"
+  ALPAKA_ACC_ANY_BT_OACC_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_CUDA_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_CUDA_ONLY_MODE: "OFF"
+  ALPAKA_ACC_GPU_HIP_ENABLE: "OFF"
+  ALPAKA_ACC_GPU_HIP_ONLY_MODE: "OFF"
+  # if OFF, run tests
+  ALPAKA_CI_ANALYSIS: "OFF"
+  # needs to be enabled, that test on the GPU are executed
+  ALPAKA_FORCE_RUNTIME_TEST: "ON"
+  ALPAKA_CI_SANITIZERS: ""
+  ALPAKA_CI_CMAKE_DIR: "/usr/local/"
+  ALPAKA_CI_INSTALL_CUDA: "OFF"
+  ALPAKA_CI_INSTALL_HIP: "OFF"
+
+gcc:
+  image: registry.gitlab.com/hzdr/crp/alpaka-group-container/alpaka-ci-gcc:1.4
+  variables:
+    CC: gcc-9
+    CXX: g++-9
+    ALPAKA_CI_GCC_VER: 9
+    ALPAKA_CI_STDLIB: libc++
+    CMAKE_BUILD_TYPE: Release
+    OMP_NUM_THREADS: 8
+    ALPAKA_DEBUG: 0
+    BOOST_ROOT: "/opt/boost/1.74.0"
+    BOOST_LIBRARYDIR: "/opt/boost/1.74.0/lib"
+    ALPAKA_CI_BOOST_LIB_DIR: "/opt/boost/1.74.0"
+    ALPAKA_CI_BUILD_JOBS: 8
+  script:
+    - source ./script/run.sh
+  tags:
+    - x86_64
+
+cuda:
+  image: registry.gitlab.com/hzdr/crp/alpaka-group-container/alpaka-ci-cuda113:1.4
+  variables:
+    CC: gcc
+    CXX: g++
+    ALPAKA_CI_GCC_VER: 7
+    ALPAKA_CI_STDLIB: libc++
+    CMAKE_BUILD_TYPE: Release
+    OMP_NUM_THREADS: 2
+    ALPAKA_DEBUG: 0
+    BOOST_ROOT: "/opt/boost/1.74.0"
+    BOOST_LIBRARYDIR: "/opt/boost/1.74.0/lib"
+    ALPAKA_CI_BOOST_LIB_DIR: "/opt/boost/1.74.0"
+    ALPAKA_CI_BUILD_JOBS: 2
+    ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE: "OFF"
```

Comment:
IMO we should work with base templates for CUDA X and use here only `extends:` to change e.g boost, ...